app/metrics.yaml

```diff
@@ -8,18 +8,12 @@ no_lint:
   - CATEGORY_GENERIC
 
 events:
-  app_received_intent:
-    type: event
-    description: |
-      The system received an Intent for the HomeActivity. An intent
-      is received an external entity wants to the app to display
-      content. Intents can be received when the app is closed – at
-      which point the app will be opened – or when the app is
-      already opened – at which point the already open app will make
-      changes such as loading a url. This can be used loosely as a
-      heuristic for when the user requested to open the app. The
-      HomeActivity encompasses the home screen and browser screen but
-      may include other screens. This differs from the app_opened
+  app_opened_all_startup:
+    type: event
+    description: |
+      A user opened the app to the HomeActivity. The HomeActivity
+      encompasses the home screen and browser screen but may
+      include other screens. This differs from the app_opened
```

Comment:
What other screens could this include, could you provide more information? I want to make sure that someone using this probe understands the cases where this could happen. Update this documentation if there's more information.

Do the `extra_keys` values remain the same?

Reply:
Other screens could include settings, collections? basically any fragment in the nav_graph. I will update the doc with some more info. 

`extra_keys` value are the same as for `app_received_intent`